public/app/plugins/datasource/prometheus/querybuilder/shared/QueryBuilderHints.tsx

```diff
@@ -56,7 +56,9 @@ export const QueryBuilderHints = <T extends PromLokiVisualQuery>({
                   size="sm"
                   className={styles.hint}
                 >
-                  {'hint: ' + hint.fix?.action?.type.toLowerCase().replace('_', ' ') + '()'}
+                  {datasource.type === 'loki'
```

Comment:
I think here we should make a discussion based on if we have the `hint.fix?.action?.hint` and not based on data source.